### ISSUE TITLE
Apptester crashes on unsubscribe action

### ISSUE THREAD

**Issue by ichorid** [Tribler/tribler#5362]
Though @devos50 already added a check for subscription widget visibility, this still happens. Can't reproduce it manually. Probably that's a result of race condition between `update_labels` and the apptester clicking on things.

```python
INFO:Executor:Performing action: subscribe_unsubscribe
INFO:Executor:Executing action: <tribler_apptester.actions.subscribe_unsubscribe_action.SubscribeUnsubscribeAction object at 0x0342D550>
INFO:Executor:Executing code with task id: 6a7e8a86b8
[PID:7440] 2020-06-12 14:12:12,893 - INFO - CodeExecutor(52) - Code execution with task b'6a7e8a86b8' finished:
[PID:7440] 2020-06-12 14:12:12,893 - INFO - CodeExecutor(53) - Stdout of task b'6a7e8a86b8': Execution of task 6a7e8a86b8 completed

[PID:7440] 2020-06-12 14:12:13,361 - ERROR <tribler_window:107> root.on_exception(): Traceback (most recent call last):
  File "src\tribler-gui\tribler_gui\widgets\subscriptionswidget.py", line 49, in <lambda>
  File "src\tribler-gui\tribler_gui\widgets\subscriptionswidget.py", line 39, in update_subscribe_button
KeyError: 'votes'

ERROR:Executor:********** TRIBLER CRASHED **********
ERROR:Executor:Tribler crashed after uptime of 1794.9829955101013 sec! Stack trace: Traceback (most recent call last):
  File "src\tribler-gui\tribler_gui\widgets\subscriptionswidget.py", line 49, in <lambda>
  File "src\tribler-gui\tribler_gui\widgets\subscriptionswidget.py", line 39, in update_subscribe_button
KeyError: 'votes'

```

**Comment by synctext** [comment]
```unsubscribe button```.
Just tested this with 250+ stress test clicks for a while. Could not get it to crash and reproduce. From the GUI I've tried:
- breaking the clicks per minute record (continuous unsubscribe/unsubscribe)
- continuous unsubscribe/unsubscribe, but also switching between "dicsovered and "my subscriptions" screen
- clicking around and also viewing the downloads screen, my channels tab.

RC5 on Ubuntu 18.04.2 LTS

**Comment by synctext** [comment]
```KeyError: 'votes'```
What if a channel has _zero_ votes or 0.0% popularity, least popular channel out of 1931 discovered also did not  crash anything. Hopefully we catch this race condition, but I would not let it block RC6.

**Comment by ichorid** [comment]
Humans can't click that fast, especially when they don't see the button )).  It probably happens when the apptester clicks the button *before* it is drawn.

**Comment by devos50** [comment]
There are 2-seconds delays _before_ and _after_ [clicking the heart](https://github.com/Tribler/application-tester/blob/master/tribler_apptester/actions/subscribe_unsubscribe_action.py#L25) so I doubt it is related to a race condition.

**Comment by devos50** [comment]
Reproduced it! It triggers when subscribing to/unsubscribing from a channel and clicking the back button before the request has been completed (e.g., the heart icon has changed state). We should fix this, since it might be the case that the Tribler core is slow to respond and a user clicks the back button quickly after clicking the heart button.

I have added [more debugging functionality](https://github.com/Tribler/application-tester/pull/34) to the app tester and print the channel being subscribed to/unsubscribed from. Hopefully, this will help us to catch further bugs.
